cosmos/dbt/graph.py

```diff
@@ -320,7 +320,7 @@ def load(
             if self.project.is_manifest_available():
                 self.load_from_dbt_manifest()
             else:
-                if execution_mode == ExecutionMode.LOCAL and self.profile_config:
+                if self.profile_config:
```

Comment:
Another condition we probably need to check is if the user set a path to the dbt project path (`self.project_dir`) - without this, we can't run dbt ls

Reply:
yes, I have added this check.